tests/test_expression.py

```diff
@@ -96,7 +423,7 @@ def get_context_data(
             {% component 'test'
                 var_a
                 ...my_dict
-                ...item
+                ..."{{ list|first }}"
```

Comment:
Are we reaaaaaallly sure we want see this in people's code?! :) 

The steps to understand this code:
1) "Oh, I didn't know the spread operator worked on strings! Oh wait, that's not a string."
2) "It's a list! And first returns the first item. So we're spreading an item in a list, what's that?"
3) "Oh, the list contains little dicts, and this dict doesn't get stringified because there isn't spaces around it."
4) "So this is actually just "a=1", I get it!"